### PR TITLE
[Merged by Bors] - chore(AlgebraicGeometry) add `Scheme.Hom.appTop`

### PR DESCRIPTION
**Pull request by erdOne**
to hide the defeq abuse `f ⁻¹ᵁ ⊤ = ⊤`

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by erdOne** [comment]
!bench

**Comment by erdOne** [comment]
!bench

File: Mathlib/AlgebraicGeometry/AffineSpace.lean [hunk 1]
```diff
@@ -122,16 +122,16 @@ lemma homOfVector_toSpecMvPoly :
     homOfVector f v ≫ toSpecMvPoly n S = (toSpecMvPolyIntEquiv n).symm v :=
   pullback.lift_snd _ _ _
 
-@[local simp]
+@[simp]
 lemma homOfVector_app_top_coord (i) :
-    (homOfVector f v).app ⊤ (coord S i) = v i := by
+    (homOfVector f v).appTop (coord S i) = v i := by
```
Comment (chrisflav) [inline Mathlib/AlgebraicGeometry/AffineSpace.lean:127]: Name should be `homOfVector_appTop_coord` and come with a deprecation.

File: Mathlib/AlgebraicGeometry/AffineSpace.lean [hunk 2]
```diff
@@ -214,13 +209,13 @@ def isoOfIsAffine [IsAffine S] :
 
 @[simp]
 lemma isoOfIsAffine_hom_app_top [IsAffine S] :
-    (isoOfIsAffine n S).hom.app ⊤ =
-      (Scheme.ΓSpecIso _).hom ≫ eval₂Hom ((𝔸(n; S) ↘ S).app ⊤) (coord S) := by
+    (isoOfIsAffine n S).hom.appTop =
+      (Scheme.ΓSpecIso _).hom ≫ eval₂Hom ((𝔸(n; S) ↘ S).appTop) (coord S) := by
```
Comment (chrisflav) [inline Mathlib/AlgebraicGeometry/AffineSpace.lean:213]: Name should be `_hom_appTop`

Comment (chrisflav) [reply Mathlib/AlgebraicGeometry/AffineSpace.lean:213]: (and below)

File: Mathlib/AlgebraicGeometry/GammaSpecAdjunction.lean [hunk 3]
```diff
@@ -415,12 +415,12 @@ theorem Scheme.toSpecΓ_base (X : Scheme.{u}) (x) :
...
 @[reassoc (attr := simp)]
 theorem Scheme.toSpecΓ_naturality {X Y : Scheme.{u}} (f : X ⟶ Y) :
-    f ≫ Y.toSpecΓ = X.toSpecΓ ≫ Spec.map (f.app ⊤) :=
+    f ≫ Y.toSpecΓ = X.toSpecΓ ≫ Spec.map (f.appTop) :=
   ΓSpec.adjunction.unit.naturality f
 
 @[simp]
 theorem Scheme.toSpecΓ_app_top (X : Scheme.{u}) :
-    X.toSpecΓ.app ⊤ = (Scheme.ΓSpecIso Γ(X, ⊤)).hom := by
+    X.toSpecΓ.appTop = (Scheme.ΓSpecIso Γ(X, ⊤)).hom := by
```
Comment (chrisflav) [inline Mathlib/AlgebraicGeometry/GammaSpecAdjunction.lean:423]: Name should be `toSpecΓ_appTop`

**Review by chrisflav — COMMENTED** [review]
Looks good, some leftovers.

**Comment by joelriou** [comment]
Thanks!

bors merge

**Pull request closed without merging**